rpcs3/Emu/Cell/lv2/sys_fs.cpp

```diff
@@ -1928,29 +1928,18 @@ error_code sys_fs_fcntl(ppu_thread& ppu, u32 fd, u32 op, vm::ptr<void> _arg, u32
 
 		const cfg::device_info device = g_cfg_vfs.get_device(g_cfg_vfs.dev_usb, vpath);
 
-		if (device.path.empty())
+		if (device.path.empty() || device.vid.empty() || device.pid.empty())
 		{
+			sys_fs.error("sys_fs_fcntl(0xc0000015): device not configured (%s)", vpath);
 			arg->out_code = CELL_ENOTSUP;
 			break;
 		}
 
 		u16 vid{};
-		{
-			auto [ptr, err] = std::from_chars(device.vid.data(), device.vid.data() + device.vid.size(), vid, 16);
-			if (err != std::errc())
-			{
-				fmt::throw_exception("Failed to read hex string: %s", std::make_error_code(err).message());
-			}
-		}
+		std::from_chars(device.vid.data(), device.vid.data() + device.vid.size(), vid, 16);
```

Comment:
Replace the 2 `fmt::throw_exception("Failed to read hex string: %s", std::make_error_code(err).message());` with 
```
sys_fs.error("Failed to read hex string: %s (returning ENOTSUP)", std::make_error_code(err).message());
arg->out_code = CELL_ENOTSUP;
break;
```
These are the only changes relevant for this pr.

Reply:
Thanks for the suggestion.
I'll do it when I'm home.

Reply:
@elad335 If I understand correctly, you wish to keep the error log, right?
The code you suggested still produces this kind of error message, which you referred to as "spamming like hell".

Reply:
How does the expected behavior of your suggested code differ from behavior of the current PR code?

Reply:
So, do you think it is fine to eliminate the error log output in this PR as per @clienthax 's suggestion?

Reply:
What was silent before this pull request should still be silent.

Reply:
Yeah, in this PR, I just added more device checks (device.vid.empty() || device.pid.empty())) and made the error messages more clear so that users can understand what's wrong more easily.

Original error log output (it blocked the thread):
SIG: Thread terminated due to fatal error: Failed to read hex string: invalid argument
(in file C:\Users\ContainerAdministrator\AppData\Local\Temp\cirrus-ci-build\rpcs3\Emu\Cell\lv2\sys_fs.cpp:1942[:25], in function sys_fs_fcntl)

New error log output in this PR(it didn't block the thread as it is not a fatal error):
sys_fs_fcntl(0xc0000015): device not configured (/dev_usbXXX)

Reply:
@elad335 Code updated as per your request.